frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -208,3 +208,56 @@ export type FirehoseResult<
 export type FirehoseResourcesResult = {
   [key: string]: FirehoseResult<K8sResourceCommon | K8sResourceCommon[]>;
 };
+
+export type PrometheusAlert = {
```

Comment:
All these new types in `console-types` don't seem to be used? Remove them if part of another PR.

Reply:
This is being used in a different file. All these are introduced to support one type `Alert`. I am moving them to the file where it's being used and also removing `export` from it.